rosbridge_library/src/rosbridge_library/internal/message_conversion.py

```diff
@@ -208,10 +208,7 @@ def _from_inst(inst, rostype):
 
     # Check for time or duration
     if rostype in ros_time_types:
-        try:
-            return {"sec": inst.sec, "nanosec": inst.nanosec}
-        except AttributeError:
-            return {"secs": inst.secs, "nsecs": inst.nsecs}
+        return {"sec": inst.sec, "nanosec": inst.nanosec}
```

Comment:
Since `rostype` is ROS 2 type, I guess there is no need to use the `secs/nsecs` format.